### PR TITLE
Master demo data independent test yti

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tivisse** [comment]
@robodoo override=ci/style

Rationale: Forward ported false-positive

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

File: addons/pos_restaurant/tests/test_frontend.py [hunk 1]
```diff
@@ -4,46 +4,64 @@
...
             'name': 'Cash 2',
             'split_transactions': False,
             'receivable_account_id': account_receivable.id,
             'journal_id': second_cash_journal.id,
         })
 
-        pos_config = self.env['pos.config'].create({
-            'name': 'Bar',
+        pos_config = cls.env['pos.config'].create({
+            'name': 'Bar Prout',
```
Comment (davidmonnom) [inline addons/pos_restaurant/tests/test_frontend.py:64]: Tf ?

Comment (tivisse) [reply addons/pos_restaurant/tests/test_frontend.py:64]: It's an energy unit of measure. See: https://en.wikipedia.org/wiki/Prout_(unit)

Comment (tde-banana-odoo) [reply addons/pos_restaurant/tests/test_frontend.py:64]: Some kind of "Atomic Bar", but more specialized. Good finding.

**Pull request closed without merging**